test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -944,16 +881,17 @@ describe('alpha router integration', () => {
 
           await validateSwapRoute(quote, quoteGasAdjusted, tradeType, 100, 10);
 
-          await validateExecuteSwap(
-            SwapType.SWAP_ROUTER_02,
-            quote,
-            tokenIn,
-            tokenOut,
-            methodParameters,
-            tradeType,
-            100,
-            100
-          );
+          // We can't actually execute a swap against mainnet without an EAT
```

Comment:
Probably worth adding this comment everywhere the `validateExecuteSwap` is commented out for clarity